Add page on citing dandisets (#159)
dandi/handbook: docs/citing.md

Citing a Dandiset
-----------------

> If you use a Dandiset in your research, please acknowledge the Dandiset by citing it, just as you would a publication,

Reviewer: ```suggestion
If you use a Dandiset or any other dataset in your research, please acknowledge it by proper citing the dataset, just as you would a publication,
```

including the DOI.
The DOI can be found in the Dandiset's landing page on the DANDI Archive website.

> An example formatted citation can also be found on the Dandiset's landing page at the "CITE AS" button. This citation

Reviewer 2: Perhaps a screenshot of an example DLP with arrows to the DOI and CITE AS locations would be helpful?

uses the DataCite citation style, which is a widely accepted standard for citing datasets, but you may need to adapt it
to the citation style required by the journal you are submitting to.

> If the Dandiset has an associated publication, it may also be appropriate to cite the publication, but this does not
> replace the need to cite the Dandiset itself.

Reviewer: ```suggestion
**If the Dandiset has an associated publication, it may also be appropriate to cite the publication, but this does not
replace the need to cite the Dandiset itself.**
```

> Citing the Dandiset is important because it provides a direct link to the data used in your research, which can be used
> to verify your results and to facilitate reproducibility. It also provides credit to the data contributors and helps
> us track the impact of the data DANDI hosts.

Reviewer: ```suggestion
Citing the Dandiset and other datasets is important because it provides a direct link to the data used in your research. That is crucial, because it:
  - allows others to better understand and verify your results, and facilitates reproducibility, 
  - connects your work to other research using the same dataset,
  - provides credit to the data collectors and maintainers, 
  - helps track the impact of DANDI and other data archives.
```

Data availability statements
----------------------------
It is common for journals to require a Data Availability Statement in the manuscript, which should include the DOI of
the Dandiset used in the research. Here is an example of a well formatted Data Availability Statement:

```
The data that support the findings of this study are openly available on the DANDI Archive at [DOI of Dandiset].
```

> It is important to note that a Data Availability Statement does not replace the need for a full citation in the
> manuscript's references section.

Reviewer: ```suggestion
**It is important to note that a Data Availability Statement does not replace the need for a full citation in the
manuscript's references section.**
```

> Citing a Dandiset

Reviewer 3: should be added to TOC/nav bar somewhere

If you use a Dandiset in your research, please acknowledge the Dandiset by citing it, just as you would a publication,
including the DOI.
The DOI can be found in the Dandiset's landing page on the DANDI Archive website.
An example formatted citation can also be found on the Dandiset's landing page at the "CITE AS" button. This citation
uses the DataCite citation style, which is a widely accepted standard for citing datasets, but you may need to adapt it
to the citation style required by the journal you are submitting to.

**If the Dandiset has an associated publication, it may also be appropriate to cite the publication, but this does not
replace the need to cite the Dandiset itself.**

Citing the Dandiset and other datasets is important because it provides a direct link to the data used in your research. That is crucial, because it:
  - allows others to better understand and verify your results, and facilitates reproducibility, 
  - connects your work to other research using the same dataset,
  - provides credit to the data collectors and maintainers, 
  - helps track the impact of DANDI and other data archives.

Data availability statements
----------------------------

> It is common for journals to require a Data Availability Statement in the manuscript, which should include the DOI of
> the Dandiset used in the research. Here is an example of a well formatted Data Availability Statement:

Author: ```suggestion
It is common for journals to require a Data Availability Statement in the manuscript, which should include the
DANDI Archive RRID and the DOI of the Dandiset used in the research.
Here is an example of a well formatted Data Availability Statement:
```

```

> The data that support the findings of this study are openly available on the DANDI Archive at [DOI of Dandiset].

Author: ```suggestion
The data that support the findings of this study are openly available on the DANDI Archive (RRID:SCR_017571) at [DOI of Dandiset].
```
